--- gtkgl.m4~	Sun Jul 30 20:06:16 2000
+++ gtkgl.m4	Wed Jul  3 22:08:23 2002
@@ -6,7 +6,7 @@
     AC_REQUIRE([AM_PATH_GTK])
     
     AC_ARG_WITH(gtkgl-prefix, [  --with-gtkgl-prefix=DIR prefix where GtkGLArea is installed ])
-    if test -n "${with_gtkgl_pefix}"; then
+    if test -n "${with_gtkgl_prefix}"; then
       gtkgl__Idir="-I${with_gtkgl_prefix}/include"
       gtkgl__Ldir="-L${with_gtkgl_prefix}/lib"
     fi
